Libraries/test/TestServerlessApp/output.template

```diff
@@ -0,0 +1 @@
ut":30,"Policies":["AWSLambdaBasicExecutionRole"],"Events":{"RootPost":{"Type":"HttpApi","Properties":{"Path":"/ComplexCalculator/Subtract","Method":"POST","PayloadFormatVersion":"2.0"}}},"PackageType":"Image","ImageUri":"626492997873.dkr.ecr.us-west-2.amazonaws.com/testserverlessapp:greetersayhello-c013811cde6a-latest","ImageConfig":{"Command":["TestServerlessApp::TestServerlessApp.ComplexCalculator_Subtract_Generated::Subtract"]}}},"PI":{"Type":"AWS::Serverless::Function","Metadata":{"Tool":"Amazon.Lambda.Annotations"},"Properties":{"MemorySize":256,"Timeout":30,"Policies":["AWSLambdaBasicExecutionRole"],"PackageType":"Image","ImageUri":"626492997873.dkr.ecr.us-west-2.amazonaws.com/testserverlessapp:greetersayhello-c013811cde6a-latest","ImageConfig":{"Command":["TestServerlessApp::TestServerlessApp.SimpleCalculator_Pi_Generated::Pi"]}}},"Random":{"Type":"AWS::Serverless::Function","Metadata":{"Tool":"Amazon.Lambda.Annotations"},"Properties":{"MemorySize":256,"Timeout":30,"Policies":["AWSLambdaBasicExecutionRole"],"PackageType":"Image","ImageUri":"626492997873.dkr.ecr.us-west-2.amazonaws.com/testserverlessapp:greetersayhello-c013811cde6a-latest","ImageConfig":{"Command":["TestServerlessApp::TestServerlessApp.SimpleCalculator_Random_Generated::Random"]}}},"Randoms":{"Type":"AWS::Serverless::Function","Metadata":{"Tool":"Amazon.Lambda.Annotations"},"Properties":{"MemorySize":256,"Timeout":30,"Policies":["AWSLambdaBasicExecutionRole"],"PackageType":"Image","ImageUri":"626492997873.dkr.ecr.us-west-2.amazonaws.com/testserverlessapp:greetersayhello-c013811cde6a-latest","ImageConfig":{"Command":["TestServerlessApp::TestServerlessApp.SimpleCalculator_Randoms_Generated::Randoms"]}}},"GreeterSayHelloAsync":{"Type":"AWS::Serverless::Function","Metadata":{"Tool":"Amazon.Lambda.Annotations","SyncedEvents":["RootGet"]},"Properties":{"MemorySize":256,"Timeout":50,"Policies":["AWSLambdaBasicExecutionRole"],"PackageType":"Image","Events":{"RootGet":{"Type":"HttpApi","Properties":{"Path":"/Greeter/SayHelloAsync","Method":"GET","PayloadFormatVersion":"1.0"}}},"ImageUri":"626492997873.dkr.ecr.us-west-2.amazonaws.com/testserverlessapp:greetersayhello-c013811cde6a-latest","ImageConfig":{"Command":["TestServerlessApp::TestServerlessApp.Greeter_SayHelloAsync_Generated::SayHelloAsync"]}}},"ToUpper":{"Type":"AWS::Serverless::Function","Metadata":{"Tool":"Amazon.Lambda.Annotations"},"Properties":{"Runtime":"dotnet6","CodeUri":"s3://normj-west2/","MemorySize":256,"Timeout":30,"Policies":["AWSLambdaBasicExecutionRole"],"PackageType":"Zip","Handler":"TestServerlessApp::TestServerlessApp.Sub1.Functions_ToUpper_Generated::ToUpper"}}},"Outputs":{"RestApiURL":{"Description":"Rest API endpoint URL for Prod environment","Value":{"Fn::Sub":"https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod"}},"HttpApiURL":{"Description":"HTTP API endpoint URL for Prod environment","Value":{"Fn::Sub":"https://${ServerlessHttpApi}.execute-api.${AWS::Region}.amazonaws.com"}}}}
```

Comment:
Should this exist?

serverless.template must have the new function.

Reply:
Removed